CONTRIBUTING.md

```diff
@@ -0,0 +1,254 @@
+
+```console
+$ cd haystack-core-integrations/integrations/chroma
+$ hatch version
+0.13.1.dev30+g9fe18d23
+```
+
+### Working on integrations
+
+> [!NOTE]
+> This is a so-called monorepo, meaning that this single repository contains multiple different applications. In our case,
+> Haystack Integrations live in the `./integrations` folder, and each one must be considered an independent project.
+> All the examples shown in this section must be executed in one specific integration folder.
+
+Hatch provides several commands we use for developing integrations, let's pick `haystack-chroma` to show a practical
+example. Before proceeding,  from the root of the repository we `cd` into the integration folder:
+
+```console
+$ cd integrations/chroma
+```
+
+#### Integration version
+
+We can get the current version of the integration with:
+
+```console
+$ hatch version
+0.13.1.dev30+g9fe18d23
+```
+
+> [!NOTE]
+> Even when you see the version number ends with a suffix like `.dev...`, this doesn't necessarily mean the integration
+> has unreleased commits. In other words, despite what the version number says, you might be looking at the latest
+> released code. This is a side effect of working with a monorepo: hatch can't tell if a certain commit in the git
+> history is releated to one integration or the other.
+
+#### Run the linter
+
+Every time you change the code, it's a good practice to run the linter, that will ensure your code is well formatted
+and the static checker is happy with the typing:
+
+```console
+$ hatch run lint:all
+```
+
+If you see Hatch reporting problems, you can try fixing them with:
+
+```console
+$ hatch run lint:fmt
+```
+
+You can also run the code formatter and the static checker separated:
+
+```console
+$ hatch run lint:style
+```
+
+and
+
+```console
+$ hatch run lint:typing
+```
+
+#### Run the tests
+
+It's important your tests pass before contributing code. To run all the tests locally:
+
+```console
+$ hatch run test
+```
+
+> [!IMPORTANT] The command above will run ALL the tests, including integration tests; some of those often need you to
+> run a certain service in background (e.g. a Vector Database) or provide credentials to external services (e.g. OpenAI)
+> in order to pass.
+
+If you want to run only a portion of the tests, for example including integration tests, Hatch will happily take the
+same options you would pass to `pytest` directly, in this case a marker with the option `-m`:
+
+```console
+$ hatch run test -m"not integration"
+```
+
+### Improving The Documentation
+
+There are two types of documentation for this project:
+
+1. Python API docs, and
+2. Documentation pages
+
+The Python API docs detail the source code: classes, functions and parameters that implement every integration.
+This type of documentation is extracted from the source code itself, and contributors should pay attention when they
```

Comment:
Do we also want to explain that pydoc config files are needed to display the docs on the website when a new integration is added?

Reply:
Yes let's do it